k8s/nginx/README.md

```diff
@@ -0,0 +1,21 @@
+# NGINX as Reverse Proxy for Mainflux services
```

Comment:
As far as i can see, this is already a part of `k8s/README.md`. Hence, please remove it.